app/src/main/kotlin/com/hippo/ehviewer/Settings.kt

```diff
@@ -152,6 +152,7 @@ object Settings : DataStorePreferences(null) {
     var harmonizeCategoryColor by boolPref("harmonize_category_color", true)
     var preloadThumbAggressively by boolPref("preload_thumb_aggressively", false)
     var downloadOriginImage by boolPref("download_origin_image", false)
+    var saveAsCbz by boolPref("save_as_cbz", true)
```

Comment:
This should default to be `false` as it introduce considerable overhead

Reply:
The archive process is actually pretty fast, but I agree we shouldn't make it default to true until 1-2 releases.

Reply:
I mean, we have not implement zero-copy read optimization for these 0-compress archives, will raise much memory pressure

Reply:
Yeah I mean until we optimize it in the next 1-2 releases.